src/equations/equations.jl

```diff
@@ -376,6 +376,12 @@ of the correct length `nvariables(equations)`.
 """
 function energy_internal end
 
+# Default implementation of derivation for `variable`. Used for subcell limiting.
+# Implementing a derivative function for a specific function improves the performance.
+@inline function variable_derivative(variable, u, equations)
```

Comment:
Not the biggest fan of `variable_derivative` since it sounds to me like "the derivative *is* variable", not that it's "the derivative _of the_ variable".

Since it's not a derivative but rather a gradient, what about something like `gradient_dxdu` or `gradient_u`?

Reply:
Since `x` already exists in this function as the spatial coordinate, I will stick with `gradient_u`. Thanks :+1:

Reply:
Done in [b776b6f](https://github.com/trixi-framework/Trixi.jl/pull/1738/commits/b776b6f226a05ec2bb6fdcfa5ef10f74c91093f9).